src/tables.ts

```diff
@@ -427,33 +427,31 @@ const articulations: Record<string, ArticulationStruct> = {
   'a,': { code: '\ue805' /*pictChokeCymbal*/, betweenLines: false }, // Choked
 };
 
-const ornaments: Record<string, { code: string }> = {
-  mordent: { code: 'ornamentShortTrill' },
-  mordentInverted: { code: 'ornamentMordent' },
-  mordent_inverted: { code: 'ornamentMordent' }, // backwards-compatible to 4.x
-  turn: { code: 'ornamentTurn' },
-  turnInverted: { code: 'ornamentTurnSlash' },
-  turn_inverted: { code: 'ornamentTurnSlash' }, // backwards-compatible to 4.x
-  tr: { code: 'ornamentTrill' },
-  upprall: { code: 'ornamentPrecompSlideTrillDAnglebert' },
-  downprall: { code: 'ornamentPrecompDoubleCadenceUpperPrefix' },
-  prallup: { code: 'ornamentPrecompTrillSuffixDandrieu' },
-  pralldown: { code: 'ornamentPrecompTrillLowerSuffix' },
-  upmordent: { code: 'ornamentPrecompSlideTrillBach' },
-  downmordent: { code: 'ornamentPrecompDoubleCadenceUpperPrefixTurn' },
-  lineprall: { code: 'ornamentPrecompAppoggTrill' },
-  prallprall: { code: 'ornamentTremblement' },
-  scoop: { code: 'brassScoop' },
-  doit: { code: 'brassDoitMedium' },
-  fall: { code: 'brassFallLipShort' },
-  doitLong: { code: 'brassLiftMedium' },
-  fallLong: { code: 'brassFallRoughMedium' },
-  bend: { code: 'brassBend' },
-  plungerClosed: { code: 'brassMuteClosed' },
-  plungerOpen: { code: 'brassMuteOpen' },
-  flip: { code: 'brassFlip' },
-  jazzTurn: { code: 'brassJazzTurn' },
-  smear: { code: 'brassSmear' },
+const ornaments: Record<string, string> = {
+  mordent: '\ue56c' /*ornamentShortTrill*/,
+  mordentInverted: '\ue56d' /*ornamentMordent*/,
```

Comment:
I thought Aaron had raised concerns about removing support for mordent_inverted and turn_inverted? If this breaks his app again we might have to reintroduce. :-)